src/main/java/life/mosu/mosuserver/domain/inquiry/InquiryJpaEntity.java

```diff
@@ -51,4 +51,8 @@ public InquiryJpaEntity(final String title, final String content, final Long use
     public void updateStatusToComplete() {
         this.status = InquiryStatus.COMPLETED;
     }
+
+    public void updateStatusToPending() {
+        this.status = InquiryStatus.PENDING;
+    }
```

Comment:
```suggestion
    public void updateStatusBy(InquiryStatus status){
         this.status = status
    }
```
로 하고, Domain Service 에서 이를 구체화해도 좋을거 같아요!

Reply:
음 저 부분이 살짝 고민되었는데 메서드 행동을 명시적으로 보여주는 게 좋을지 아니면 상태 값을 받아서 변경할 지 
태현님은 혹시 이 부분에 대해서 어떻게 생각하시나요?